source/sass/main.scss

```diff
@@ -31,6 +31,10 @@ $bp-xl: #{map-get($grid-breakpoints, xl)}; // >= 1200px
 @import '../js/components/fellow-list/fellow-list';
 @import '../js/components/petition/petition';
 
+// Special react component
+
+@import '../js/donate-modal/donate-modal';
```

Comment:
![screenshot_2018-11-21 homepage](https://user-images.githubusercontent.com/2896608/48883443-b3c9da00-edd4-11e8-9995-22ac9e5ec11c.png)

We should add the following rules to `.btn-donate` in `donate-modal.scss`

```css
background: #ff4f5e;
border-radius: 14px;
font-size: 14px;
```

Reply:
done, with a `color: white` added, too.